prombench/manifests/prombench/stateful-set.yaml

```diff
@@ -70,13 +70,13 @@ spec:
               - "/bin/sh"
               - "-c"
               - "rm /tmp/READY && until make all_nodepools_running; do echo waiting for nodepools to be created; sleep 10; done; make clean; kill -9 -1"
-              # wait until all_nodepools_running returns a success code then run make clean and kill the container
+              # Remove ready status, wait until all_nodepools_running returns a success code then run make clean and kill the container.
```

Comment:
Would you mind putting the comment above the command to make it consistent with the other places and also it is the most common way of commenting.